docs/source/cpp_api/mdspan.rst

```diff
@@ -216,13 +222,10 @@ Device Vocabulary
 Device Factories
 ----------------
 
-.. doxygenfunction:: raft::make_device_mdspan
-    :project: RAFT
-
 .. doxygenfunction:: raft::make_device_matrix_view
     :project: RAFT
 
-.. doxygenfunction:: raft::make_device_vector_view
+.. doxygenfunction:: raft::make_device_vector_view(ElementType* ptr, IndexType n)
```

Comment:
This might be a dumb question - but why do we need to update this function to include the parameters, but not the make_device_matrix_view function right above it (or make_device_scalar_view below it)?  It seems we should be consistent here

Reply:
Mostly to keep this consistent with the other APIs. doxygen has been getting really confused with some overloads, even when the overloads aren't necessarily ambiguous so I've been specifying the params in the meantime. Another reason I've been specifying the params is so that we can isolate the mdspan-based APIs in the docs and ignore the pointer-based APIs in the meantime.

I'm hoping we can eventually find a better way to handle this.